features/fixtures/TestApp/config/config.yml

```diff
@@ -45,11 +45,11 @@ fos_user:
     firewall_name:    api
     service:
         user_manager: api.user_manager
-    user_class:       "Dunglas\ApiBundle\Tests\Behat\TestBundle\Entity\User"
+    user_class:       "Dunglas\\ApiBundle\\Tests\\Behat\\TestBundle\\Entity\\User"
```

Comment:
I remember correctly: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Yaml/CHANGELOG.md

It would be easier to read with single quotes, what do you think?